### DISCUSSION TITLE
How do I make the bot render new lines?

### DISCUSSION THREAD

**Discussion by MrCordeiro** [tjtanjin/react-chatbotify#65] [Q&A] [answered]
I'm connecting the chatbot to an LLM that streams messages with `\n\n` to split between different paragraphs. The chatbot, however, renders those as a single space.

So the result is a long wall of test. How can I configure the chat so that line breaks are rendered?

### Things I tried unsuccessfully 

- Replacing  `\n\n` with `<br/><br/>` on the backend and then setting `botBubble: { dangerouslySetInnerHTML: true }`

**Comment by MrCordeiro** [comment]
Never mind, I had a typo!  (o′┏▽┓｀o)

**Comment by tjtanjin** [comment]
Hey @MrCordeiro,

Coincidentally, this was addressed in a commit made just a few days ago. I'm actually working on getting out v1.6.2 tonight or tomorrow. With that said you wouldn't need to do manual replacements of `\n` anymore.

**Comment by tjtanjin** [comment, marked as answer]
Hey @MrCordeiro, v1.6.2 is out now and you can give it a try 😊 Let me know if there're still issues!

**Reply by MrCordeiro** [reply]
YES!!! It worked! **Thank you!**
Now that's what I call issue resolution SPEED! 🏁🏎️
